src/node_options.cc

```diff
@@ -1,8 +1,9 @@
-#include <errno.h>
```

Comment:
Is it unrelated? We usually include the system headers first so that we are able to override things with our own headers.

Reply:
AFAIK re include system headers last so that our own headers don't become dependent on them by mistake.

Ref: https://github.com/nodejs/node/pull/26042/files/fdd0d8e0e07413e98e9da78f473fb15aa7fe24f7#diff-adab18a152106965b50519ebab33e087

Reply:
Yeah, I’d also include system headers last unless we need to do it another way.

But since it sounds like this is unrelated, can you split it out into separate PRs/commits?